### PR TITLE
RHPAM-3606 - Unknown Custom tasks in Designer makes Diagram Explorer empty

### PR DESCRIPTION
**Pull request by Josephblt**
If an item could not be found in the workiItemDefinitionRegistry a null exception was thrown interrupting the load process.

**JIRA**: [RHPAM-3606](https://issues.redhat.com/browse/RHPAM-3606)

**Referenced Pull Requests**:
* https://github.com/kiegroup/kogito-editors-java/pull/36

**Business Central**: [WAR file - Not generated yet](https://donwnload.here/something)

### CODE REVIEW

**Comment by domhanak** [comment]
@Josephblt Could you please assign reviewers? Is this ready for review? Why are the builds not run?

**Comment by Josephblt** [comment]
Jenkins run cdb
Jenkins run fdb

**Comment by domhanak** [comment]
I ll take a look when FDB's are done

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-project-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java [hunk 1]
```diff
@@ -118,15 +131,25 @@ public void testViewHandler() {
         when(imageProxy.isLoaded()).thenReturn(true);
         tested.viewHandler().accept(task,
                                     view);
-        verify(imageProxy, times(1)).load(eq(WID_ICON_DATA));
+        verify(imageProxy, times(1)).load(eq(WID_ICON_DATA_1));
     }
 
     @Test
     public void testGetGlyph() {
```
Comment (domhanak) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-project-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java:138]: Could we split this into multiple test cases?

**Review by domhanak — APPROVED** [review]
The issue is fixed. I was not able to reproduce the issue in BC. Consider my comment please.Thanks!

**Review by romartin — APPROVED** [review]

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-project-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java (hunk 1 above)
Comment (romartin) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-project-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java:138]: hey @Josephblt @domhanak , IMO the test cases are quite short and easy to manage and understand, so what @domhanak mentions make sense, although I don't see a blocker for this PR. Anyway guys, lemme know once you agree on thsi comment and I'll just merge. THanks!

Comment (Josephblt) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-project-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java]: Done. Thanks @domhanak.

**Comment by Josephblt** [comment]
Jenkins run cdb
Jenkins run fdb

**Comment by Josephblt** [comment]
Jenkins run fdb
Jenkins retest this

**Pull request merged**